plugins/auditlib/CMakeLists.txt

```diff
@@ -24,20 +24,32 @@
 
 project( auditlib ) 
 
-set (    SRCS 
-         auditlib.cpp 
-    )
-
-include_directories ( 
-         ./../../system/include 
-         ./../../system/jlib 
-    )
-
-ADD_DEFINITIONS( -D_USRDLL -DAUDITLIB_EXPORTS )
-
-HPCC_ADD_LIBRARY( auditlib SHARED ${SRCS} )
-install ( TARGETS auditlib DESTINATION plugins )
-target_link_libraries ( auditlib  
-         jlib 
-    )
-
+if(AUDITLIB)
+    ADD_PLUGIN(auditlib)
+    if(MAKE_AUDITLIB)
```

Comment:
question: What is the difference between AUDITLIB and MAKE_AUDITLIB?

Reply:
It seemed to be the common pattern, The MACROs allow you to exclude all except the one you want to include or include all except the ones you suppress.